src/example.jl

```diff
@@ -1,32 +1,25 @@
-""" The Academic Performance Index is computed for all California schools based on standardised testing of students. The data sets contain information for all schools with at least 100 students and for various probability samples of the data.
+const PKG_DIR = joinpath(pathof(Survey), "..", "..") |> normpath
+asset_path(file) = joinpath(PKG_DIR, "assets", file)
 
-The ```data(api)``` function loads 3 dataframes: apiclus1, apiclus2, apipop.
+"""
+The Academic Performance Index is computed for all California schools based on standardised testing of students.
+The data sets contain information for all schools with at least 100 students and for various probability samples of the data.
+
+Use `load_data(api)` to load API data. 
+
+```julia
+# load all five dataframes
+(; apiclus1, apiclus2, apipop, apistrat, apisrs) = load_data(api)
```

Comment:
I think this needs to be changed according to the latest change to the `load_data` function.